docker/entrypoint_uwsgi.sh

```diff
@@ -20,6 +20,9 @@ echo "DEBUG: " $DEBUG
 echo "DJANGO_TEST_SERVER: " $DJANGO_TEST_SERVER
 echo "------------------------------"
 
+# start required daemons
+tor --DataDirectory $(mktemp -d) &
```

Comment:
honestly I do not like this to happen by default.

Tor should be started only if the user wants to use the OnionScan analyzer. Most of the time the user won't. 

This is also the reason why, in my opinion, this analyzer cannot be enabled by default. We should force the user to choose to activate it if he wants, and let him understand what he is doing. Open a port to Tor cannot just be taken so lightly.

Reply:
Right, open tor ports can serve as dangerous. Should we use the daemon from a subprocess in onionscan instead?